docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,231 @@
+Using Scaleway with Ansible
+===========================
+
+Introduction
+------------
+
+`Scaleway <https://scaleway.com>`_ is cloud provider supported by Ansible (>=2.6) via a dynamic inventory script and modules.
+Those modules are:
+
+- `scaleway_sshkey <https://docs.ansible.com/ansible/devel/modules/scaleway_sshkey_module.html>`_: adds a public SSH key from file or value to the Packet infrastructure. Every subsequently-created device will have this public key installed in .ssh/authorized_keys.
```

Comment:
Please use :ref: -style references for links to internal documentation pages. The syntax for linking to module docs is `:ref:'modulename_module'` but replace the single quotes with backticks.